package.json

```diff
@@ -54,6 +61,8 @@
     "stylelint": "^13.2.1",
     "stylelint-config-sass-guidelines": "^7.0.0",
     "vue-docgen-api": "^4.14.0",
+    "vue-jest": "^3.0.5",
+    "vue-router": "^3.1.6",
```

Comment:
The `package-lock.json` had `vue-router` at a top level until I did a fresh `npm install` for the new testing dependencies. Then it got moved to a lower child level in the package-lock, preventing access to it from all other dependencies. Not sure how it got to the top level to begin with, but anyways, without it apparently the `docs:build` fails, so I just added it as a devDep.